### ISSUE TITLE
Shadow Demon Ultimate - Demonic Purge

### ISSUE THREAD

**Issue by fdejesusjr** [ash47/LegendsOfDota#214]
With Sceptre settings on, the ultimate does not gain any charges. Does not work

**Comment by ash47** [comment]
I don't think this spell has ever worked, let alone with a scepter, will see what I can do :/

**Comment by milinz** [comment]
It seems to require charges with/without scepter atm. With scepter the charge buff appears but never gains any charges.

**Comment by fdejesusjr** [comment]
Thanks for getting back to me, I love the game and think you are doing an
amazing job, I play it all the time with my friends, I think this move
would be great to help with all those all passive right click builds I keep
coming across.

Figured I would share the info that it wasn't working, keep up the amazing
work on the game and if possible I would love to see this in action but if
not still an amazing job with everything guys. Thanks again.

I did have a thought I wanted to share just in case you were interested in
implementing it.

What if there were something along the lines of multicast runes or spell
buff runes, magic immunity, reflect damage shield etc idk it was just an
idea friends and I were talking about that we thought would be cool as temp
buffs in the game via runes.

Anyways love the game keep it up and all the best!
